clickhouse/restore-clickhouse.sh

```
@@ -0,0 +1,298 @@
+        log "Volume is empty, skipping safety backup"
+        current_backup="N/A (volume was empty)"
+    fi
+
+    # Download backup from storage box
+    log "Downloading backup from storage box..."
+    if ! scp "${STORAGE_BOX_HOST}:${BACKUP_BASE_DIR}/${backup_file}" "$temp_backup" 2>&1 | tee -a "$LOG_FILE"; then
+        rm -f "$temp_backup"
+        error "Download failed"
+    fi
+
+    local archive_size=$(du -h "$temp_backup" | cut -f1)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)